requirements-dev.txt

```diff
@@ -32,3 +32,4 @@ git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e
 xgboost>=1.3
 tqdm
 modin-spreadsheet>=0.1.1
+pymssql
```

Comment:
Should we have this in setup.py too?

Reply:
I think setup.py was for all users, not just developers. The new module is only necessary for users who want to read from SQL server with pymssql, so I think it doesn't need to go to setup.py.